online_catalog/TRLN-Discovery/ICPSR/forge-icpsr-codebook2-0-draft.xsl

```diff
@@ -94,9 +94,9 @@
     "publisher": [
         "<xsl:value-of select="normalize-space($producer)"/>"
     ],
-    "resource_type": "Dataset – Statistical",
-    "access_type": "Online",
-    "institution": ["unc", "duke", "nccu", "ncsu"],
+    "resource_type": ["Dataset –- Statistical"],
```

Comment:
These changes are probably on me (i.e. not specifying it properly to start with): 
* resource_type and access_type need to be an arrays of string values
* nccu actually doesn't have access to ICSPR datasets